packages/flutter/lib/parse_server_sdk_flutter.dart

```diff
@@ -78,7 +78,9 @@ class Parse extends sdk.Parse
       appVersion: appVersion,
       appPackageName: appPackageName,
       locale: locale ??
-          (sdk.parseIsWeb ? ui.window.locale.toString() : Platform.localeName),
```

Comment:
What is this change?

Reply:
> info • 'window' is deprecated and shouldn't be used. Look up the current FlutterView from the context via View.of(context) or consult the PlatformDispatcher directly instead. Deprecated to prepare for the upcoming multi-window support. This feature was deprecated after v3.7.0-32.0.pre • packages/flutter/lib/parse_server_sdk_flutter.dart:81:32 • deprecated_member_use
error

The `ui.window.locale` is deprecated. Should I open a new PR to describe this change?

Reply:
@Nidal-Bakir 
Its replacement has been changed as follows
```ui.window.locale``` to ```ui.PlatformDispatcher.instance.locale```

Reply:
Got it, thanks